python/setup.py

```diff
@@ -89,10 +75,10 @@ def has_ext_modules(self):
     author='DMLC',
     maintainer='Hyunsu Cho',
     maintainer_email='chohyu01@cs.washington.edu',
+    zip_safe=False,
     packages=find_packages(),
     install_requires=['numpy', 'scipy'],
-    package_data={
-        'treelite': [LIB_BASENAME, RT_BASENAME, 'treelite_runtime.zip', 'VERSION']
-    },
-    distclass=BinaryDistribution
+    data_files=[('treelite', DATA_FILES)],
+    license='Apache-2.0',
+    python_requires='>=3.4'
```

Comment:
Python 3.4 reached its end-of-life 2019-03-18.
https://devguide.python.org/devcycle/#end-of-life-branches

Reply:
Thanks!